build.gradle

```diff
@@ -0,0 +1,53 @@
+apply plugin: 'java'
+apply plugin: 'application'
+apply plugin: 'pmd'
+
+group 'com.github'
+version '1.0-SNAPSHOT'
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    implementation group: 'net.sourceforge.pmd', name: 'pmd-java', version: '6.29.0'
+    implementation 'info.picocli:picocli:4.5.2'
+
+    testImplementation(platform('org.junit:junit-bom:5.7.0'))
+    testImplementation('org.junit.jupiter:junit-jupiter')
+
+    testRuntimeOnly("org.junit.jupiter:junit-jupiter-engine:5.7.0")
+}
+
+pmd {
+    consoleOutput = true
+    ruleSetFiles = files("PMDRuleSet.xml")
+}
+
+test {
+    useJUnitPlatform()
+}
+
+application {
+    mainClassName = 'com.github.emcat.Emcat'
+    applicationDefaultJvmArgs = ["--enable-preview"]
```

Comment:
Better not to use preview features

Reply:
In further PRs I get rid of preview mode and migrate to java 11.
We can keep it here and further PRs will fix this problem.